parquet/src/arrow/async_reader/mod.rs

```diff
@@ -98,10 +98,22 @@ pub trait AsyncFileReader: Send {
         .boxed()
     }
 
-    /// Provides asynchronous access to the [`ParquetMetaData`] of a parquet file,
-    /// allowing fine-grained control over how metadata is sourced, in particular allowing
-    /// for caching, pre-fetching, catalog metadata, etc...
-    /// ArrowReaderOptions may be provided to supply decryption parameters
+    /// Return a future which results in the [`ParquetMetaData`] for this Parquet file.
+    ///
+    /// This is an asynchronous operation as it may involve reading the file
+    /// footer and potentially other metadata from disk or a remote source.
+    ///
+    /// Reading data from Parquet, requires the metadata to understand the
+    /// schema, row groups, and location of pages within the file. This metadata
+    /// is stored in the footer of the Parquet file, and can be read using
```

Comment:
```suggestion
    /// is stored primarily in the footer of the Parquet file, and can be read using
```
Page indexes are not technically in the footer. And if we add bloom filters to `ParquetMetaDataReader` at some point...